translations/bnpchs-2018-penyisihan-lampar-dadu.html

```diff
@@ -0,0 +1,50 @@
+<!--Tossing dice-->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix has two dices each with $$$N$$$ sides, he has an idea of a game with those two dices. On each side of the dice, there is a number between $$$1$$$ and $$$N$$$ inclusive and the numbers on each side of a dice is different. This game is played by 2 players, player one picks a number between $$$1$$$ and ($$$N - 1)$$$ inclusive. Then, both players toss their dice, if the maximum number that appears on the top of both dice is smaller or equal to the number picked by the first player, then the first player wins the game. If not, then the second player wins.
```

Comment:
```suggestion
  Jelix has two dice each with $$$N$$$ sides, he has an idea for a game with those two dice. On the die, there are distinct numbers between $$$1$$$ and $$$N$$$ (inclusive) for each side. This game is played by 2 players, player one picks a number between $$$1$$$ and $$$(N - 1)$$$ (inclusive). Then, both players toss their die, if the maximum number that appears on the top of both dice is smaller or equal to the number picked by the first player, then the first player wins the game. If not, then the second player wins.
```

Note: dice is plural, die is singular

Reply:
oh rlly? woah interesting